core/domain/suggestion_registry.py

```diff
@@ -1101,27 +1101,30 @@ class ReviewerAndSuggestionCounts(python_utils.OBJECT):
     """Domain object for the ReviewerAndSuggestionCountsModel.
 
     Attributes:
-        translation_reviewer_counts_per_lang: dict. A dictionary that contains
-            the total number of translation reviewers for each language that
-            translation suggestions are offered in.
-        translation_suggestion_counts_per_lang: dict. A dictionary that
-            contains the total total number of translation suggestions that are
-            currently in review per language.
+        translation_reviewer_counts_by_lang: dict. A dictionary where the keys
+            represent the languages that translation suggestions are offered in
```

Comment:
Docstring looks much better, but just wanted to get more clarity on what the keys are. Are they language names or language codes? If the latter, say that.

Reply:
Changed! Thanks :)